src/LoansPage.tsx

```diff
@@ -1,7 +1,148 @@
-import React from "react";
+import React, { useState, useEffect, useMemo } from "react";
+import { Link } from "react-router-dom";
+
+import { Loan } from "./ourtypes";
+import PageLayout from "./PageLayout";
+import AutoTable, { TableHeader } from "./Table";
+import { Button } from "react-bootstrap";
 
 const LoansPage: React.FC = () => {
-  return <div>Loans page!</div>;
+  const [reqIsLoaded, setReqIsLoaded] = useState(false);
+  const [ownIsLoaded, setOwnIsLoaded] = useState(false);
+  const [loanedToMe, setLoanedToMe] = useState<Loan[]>([]);
+  const [loanedByMe, setLoanedByMe] = useState<Loan[]>([]);
+
+  useEffect(() => {
+    fetch("http://paralibrary.digital/api/loans/requester")
+      .then((res) => {
+        setReqIsLoaded(true);
+        return res.json();
+      })
+      .then(
+        (result) => {
+          setLoanedToMe(result);
+        },
+        (error) => {
+          console.log(error);
+        }
+      )
+      .catch((error) => {
+        setReqIsLoaded(true);
```

Comment:
You could put this in a `.finally()` block so you don't have to put this in both the .then and .catch blocks